crates/tooling/multiversion-tests/examples/readme.md

```
@@ -0,0 +1,12 @@
+example set of configurations for running multiversion tests from a significantly older version (9f074ccf35319cbb046b150a2e37e4ab37feb66b) to HEAD (fd107a5e9280498ae84fcccc784ffceeeb6f8fa8)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)